notebooks/official/prediction/llm_streaming_prediction.ipynb

```diff
@@ -0,0 +1,791 @@
+{
```

Comment:
<p>pandas &gt; Pandas</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2255/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>

Reply:
pandas is not capitalized: https://pandas.pydata.org/docs/user_guide/10min.html